src/main/java/bio/terra/service/job/JobService.java

```diff
@@ -162,34 +155,22 @@ public boolean shutdown() throws InterruptedException {
       logger.info("JobService request Stairway terminate");
       finishedShutdown = stairway.terminate(terminateTimeout, TimeUnit.SECONDS);
     }
-    logger.info("JobService finished shutdown?: " + finishedShutdown);
+    logger.info("JobService finished shutdown?: {}", finishedShutdown);
     return finishedShutdown;
   }
 
   public int getActivePodCount() {
     return kubeService.getActivePodCount();
   }
 
-  public static class JobResultWithStatus<T> {
-    private T result;
-    private HttpStatus statusCode;
+  public record JobResultWithStatus<T>(HttpStatus statusCode, T result) {
```

Comment:
Looking at how this is used, `<T>` is always `<Object>` so this could be changed to a non-generic form.